samigo/samigo-app/src/webapp/js/authoring.js

```diff
@@ -389,16 +400,19 @@ function disableAllFeedbackCheckTemplate(feedbackType)
 	}
 }
 
-$(document).ready(function(){
+$(window).load( function() {
```

Comment:
I'm unclear on the logic here. Isn't document.ready the preferred way here?

http://learn.jquery.com/using-jquery-core/document-ready/